tests/conftest.py

```diff
@@ -2347,7 +2348,13 @@ def _remove_entry(table_name, key_name, config):
             }
             logger.warning("Core dump or config check failed for {}, results: {}"
                            .format(module_name, json.dumps(check_result)))
-            results = parallel_run(__dut_reload, (), {"duts_data": duts_data}, duthosts, timeout=360)
+
+            is_modular_chassis = duthosts.nodes[0].get_facts().get("modular_chassis")
```

Comment:
Nit-pick: we can just use duthosts[0].get_facts()....

Reply:
Refined, thank for remind